resources/warnings.json

```diff
@@ -11367,8 +11367,8 @@
     "url": "https://www.jenkins.io/security/advisory/2022-03-15/#SECURITY-2232",
     "versions": [
       {
-        "lastVersion": "346.vd87693c5a_86c",
-        "pattern": ".*"
+        "lastVersion": "0.56",
+        "pattern": "0\\.\\d\\d?"
```

Comment:
This makes no sense, because `0.56` is not the last release with major version `0`, so the pattern matches later versions.